### PR TITLE
Bump to Mutiny 2.5.2

### PR DESCRIPTION
**Pull request by jponge**

### CODE REVIEW

**Comment by jponge** [comment]
I'd like to hear back from the impact on Reactive Messaging before we rebase + merge this https://github.com/smallrye/smallrye-reactive-messaging/pull/2404

**Review by gsmet — DISMISSED** [review]

**Comment by jponge** [comment]
There's a race condition in the revamped `concatMap` operator of Mutiny 2.5.2. It did fix bugs, bug opened new ones 🤣 

I'll keep that PR open to prevent dependabot to request an upgrade. Until then please do not merge 👍

**Comment by jponge** [comment]
2.5.3 coming with fixes

**Pull request closed without merging**
